packages/commonwealth/client/scripts/controllers/server/newProfiles.ts

```diff
@@ -100,6 +101,10 @@ class NewProfilesController {
             const currentProfile = result.find(
               (r) => r.address === profile.address
             );
+            return {profile, currentProfile}
+          }).filter(({profile, currentProfile}) => {
+            return currentProfile !== undefined;
+          }).map(({profile, currentProfile}) => {
```

Comment:
`currentProfile` could be undefined
occured 66 times on `community>proposals` page
<img width="708" alt="image" src="https://user-images.githubusercontent.com/4791635/231207102-e767b516-0268-4f43-ac69-880577917a28.png">

Reply:
Properly fixed by 
https://github.com/hicommonwealth/commonwealth/pull/3391/files